QuizApp/App/Screens/Quiz/QuizCell.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+final class QuizCell: UICollectionViewCell {
+
+    var data: Category? {
+        didSet {
+            guard let data = data else { return }
+            update(data: data)
+        }
+    }
```

Comment:
Idea here is to expose only `data` property for to be set for updating the cell with needed info (data source). Hence, the `update` method and its implementation is private and hidden.

I mean, I can expose the `update` method and remove the property. But then, the implementation is exposed too.

Reply:
"...the implementation is exposed too." - I don't understand what you mean. 
`update` method will be public, yes. But what is the point of having it hidden? 
`data` isn't used anywhere, other than as an argument to `update` method, so it seems to me there is no point in storing it. 

Furthermore, if update method was defined as: 
`func set(for category: Category) { ... }` 

you'd call cell.set(for: category) and it would be much more intuitive to use.

Reply:
After some contemplation about the value of `data` property, and your comments, I do agree that the value of having it is not really significant, if any. I will remove it and use the proposed `set` method.